frostsnap_core/src/message.rs

```diff
@@ -198,6 +202,7 @@ pub enum DeviceToCoordinatorMessage {
         share_index: PartyIndex,
         success: bool,
     },
+    LoadBackupResult(Box<EnteredPhysicalBackup>),
```

Comment:
Always unknown, s/LoadBackupResult/LoadUnknownBackupResult/

Reply:
Maybe also the case for `LoadPhysicalBackup`? Will see how this looks after later commits

Reply:
Yes I will fix the naming here.